### PR TITLE
chore: update pill, mastercard and icons

### PR DESCRIPTION
**Pull request by burbuli8ra**
This PR includes the following:
* Update the list of icons supported by `Icon` component and add support for directory grouping
* Update cursor of `Pill` component to show as `default` when no `onClick` is provided
* Update text size to be configurable explicitly (if needed)
* Update `MasterCard` component to support `data-testid` for each `Pill`
* Update `MasterCard` component's background colors with fallbacks
* Update `MasterCard` component to support `onClick` functions for each `Pill`
* Expose `MasterCard` component from library

### CODE REVIEW

**Review by jacekkolasa — APPROVED** [review]
good stuff! I was thinking - maybe we should start grouping those icons somehow? (in this case they could go to OS group, maybe with the prefix?)

**Review by jacekkolasa — APPROVED** [review]
thx @burbuli8ra !

File: src/components/pill/container/index.js [hunk 1]
```diff
@@ -7,24 +7,24 @@ import getPillPadding from "../mixins/padding"
...
-  ({ round = 999, hollow, flavour, borderColor, padding, size, tiny, width, height }) => ({
+  ({ round = 999, hollow, flavour, borderColor, onClick, padding, size, tiny, width, height, position }) => ({
     padding: getPillPadding(padding, size, tiny),
     round,
     border: {
       side: "all",
       color: borderColor || getPillColor(hollow ? "border" : "background", flavour),
       size: "1px",
     },
+    cursor: onClick ? "pointer" : "default",
```
Comment (AlexNti) [inline src/components/pill/container/index.js:18]: Nice!

**Comment by AlexNti** [comment]
Great Job!

**Review by AlexNti — APPROVED** [review]

**Pull request merged**